addon/services/permissions.ts

```diff
@@ -1,17 +1,25 @@
 import { EVENTS } from '@bagaar/ember-permissions/-private/config';
 import { assert } from '@ember/debug';
-import { addListener, sendEvent } from '@ember/object/events';
+import { sendEvent } from '@ember/object/events';
+import Transition from '@ember/routing/-private/transition';
+import RouterService from '@ember/routing/router-service';
 import Service, { inject as service } from '@ember/service';
 
+type Permission = string;
+type UserPermissions = Permission[];
```

Comment:
I added these types for readability, but I'm not using them consistently I think. I should double check this.